backend/app/jobs/translate_job.rb

```diff
@@ -4,7 +4,7 @@ class TranslateJob < ApplicationJob
   rescue_from(ActiveRecord::RecordNotFound) do |exception|
     Rails.logger.error(exception)
     Rails.logger.error(Rails.backtrace_cleaner.clean(exception.backtrace).join("\n"))
-    # TODO: handle exception tracking
+    Google::Cloud::ErrorReporting.report exception
```

Comment:
I remember that I have added some hooks for BackgroundJob Events to ApplicationJob and they also store exception to db. Maybe that reporting should be added there and all these `rescue_from ` and `handle_exception ` could be removed so we deal with exceptions at one place for all current and future jobs.

Reply:
```
def store_exception_at_event
    yield
  rescue Exception => e # rubocop:disable Lint/RescueException
    upsert_event self, status: :crashed, message: e.backtrace
    raise e
  end
```

Reply:
I agree we can move it there.

Reply:
OK! Thanks!

Reply:
Oh but actually in that case we do a `raise` - that will automatically report

Reply:
Ohh, ok. Then, if we still need to have the clean error messages in the logger (not sure if there is any need of that now, if we have error management), we can move Rails.logger part.

Reply:
We can keep it, dunno :)

Reply:
Or actually, raising exception will get a good log as well, so we can just remove the whole thing, lol.